test/tiles/chromsizes_test.py

```diff
@@ -0,0 +1,22 @@
+import os.path as op
+
+import clodius.tiles.chromsizes as ctcs
+from clodius.models.tileset_info import TilesetInfo
+
+
+def test_get_tileset_info():
+    filename = op.join("data", "chromSizes.tsv")
+
+    # Test loading tileset info using a filename
+    tsinfo = TilesetInfo(**ctcs.tileset_info(filename))
+
+    assert tsinfo.max_width > 100
+    assert len(tsinfo.chromsizes) > 2
+    # TODO: Do something with the return value
```

Comment:
The parsing with the pydantic model is a kind of assertion here, is that what you were thinking wrt to  "return value'?

Reply:
Oh, I meant to remove that line. Removing in the next commit.